src/perf_ros2.cpp

```diff
@@ -100,7 +100,7 @@ class Perf : public rclcpp::Node, public event_camera_codecs::EventProcessor
     const double dt = (t - lastTime_).nanoseconds() * 1e-9;  // in milliseconds
     if (numMsgs_ != 0 && t_elapsed > 1e-3) {
       printf(
-        "%.0f msgs: %8.2f/s drp: %2zu del: %5.2fms drft: %8.4fs", t_elapsed, numMsgs_ / dt,
```

Comment:
This change does not compile on the CI server because it replaces one bad formatter (%2zu) with another %2llu.
The proper fix is to use the PRIu64 macro
#include <cinttypes>
uint64_t foo(0);
std::printf("The value is: %" PRIu64 "\n", foo);